packages/strapi-tiptap-editor/admin/src/components/Editor/index.tsx

```diff
@@ -60,6 +60,8 @@ const Editor = ({ editor, settings, productPrice }: EditorProps) => {
   if (!settings) {
     return null;
   }
+  type Theme = 'light' | 'dark';
+  const currentTheme = localStorage.getItem('STRAPI_THEME') as Theme;
```

Comment:
Je doet heer niet echt aan input validatie. How about:

```tsx
const availableThemes: Theme[] = ['light', 'dark'];
const localStorageTheme = localStorage.getItem('STRAPI_THEME');
const theme = availableThemes.find(x => x === localStorageTheme) || 'light';
```

Ik zou bovendien een utility functie gebruiken voor `getLocalStorage` met een `try`/`catch`, `localStorage` is een beetje flaky.